web/modules/custom/collection/collection.module

```diff
@@ -372,6 +379,30 @@ function collection_rdf_entity_view(array &$build, EntityInterface $entity, Enti
       ],
     ];
   }
+
+  // Read more link to the about page.
+  if ($display->getComponent('read_more')) {
+    $build['read_more'] = $entity->toLink(t('Read more'), 'about-page', [
+      'attributes' => [
+        'class' => ['about-page', 'read-more', 'fieldset__read-more'],
+      ],
+    ])->toRenderable();
+  }
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function collection_rdf_entity_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  /** @var \Drupal\rdf_entity\RdfInterface $entity */
+  if ($entity->bundle() !== 'collection' || $display->getMode() !== 'default') {
+    return;
+  }
+
+  // Hide the description when the abstract is available.
+  if (Element::children($build['field_ar_abstract'])) {
+    $build['field_ar_description']['#access'] = FALSE;
```

Comment:
Is there a good reason why we are not disabling the description via the entity view display? I commented this line, then I disabled the field from the entity display and I achieved the same goal. Also, I'm repeating my belief here (and I will do it again and again): We should switch, as much as we can, from maintaining code to maintaining config.

Reply:
This is done because part of the scope of the ticket is to show the description only when the abstract is not available. So we cannot use configuration here.